CoreMLHelpers/UIImage+Extensions.swift

```diff
@@ -1,45 +1,75 @@
-
-  The above copyright notice and this permission notice shall be included in
-  all copies or substantial portions of the Software.
-
-  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
-  IN THE SOFTWARE.
-*/
+ Copyright (c) 2017-2019 M.I. Hollemans
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to
+ deal in the Software without restriction, including without limitation the
+ rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ sell copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ IN THE SOFTWARE.
+ */
 
 #if canImport(UIKit)
 
 import UIKit
 
 extension UIImage {
-  /**
-    Resizes the image.
-
-    - Parameters:
-      - scale: If this is 1, `newSize` is the size in pixels.
-  */
-  @nonobjc public func resized(to newSize: CGSize, scale: CGFloat = 1) -> UIImage {
-    let format = UIGraphicsImageRendererFormat.default()
-    format.scale = scale
-    let renderer = UIGraphicsImageRenderer(size: newSize, format: format)
-    let image = renderer.image { _ in
-      draw(in: CGRect(origin: .zero, size: newSize))
+    /**
+     Resizes the image.
+     
+     - Parameters:
+     - scale: If this is 1, `newSize` is the size in pixels.
+     */
+    @nonobjc public func resized(to newSize: CGSize, scale: CGFloat = 1) -> UIImage {
+        let format = UIGraphicsImageRendererFormat.default()
+        format.scale = scale
+        let renderer = UIGraphicsImageRenderer(size: newSize, format: format)
+        let image = renderer.image { _ in
+            draw(in: CGRect(origin: .zero, size: newSize))
+        }
+        return image
+    }
+    
+        /**
+     Rotates the image.
+     
+     - Parameters:
+     - angle: Angle degree (0-360).
+     */
+    @nonobjc public func rotate(angle:CGFloat)->UIImage
```

Comment:
Please use the same formatting conventions as the original code, w.r.t spacing etc: `rotate(angle: CGFloat) -> UIImage`

This applies to the rest of this function too.

Reply:
I'd prefer `rotate(degrees:)` over `angle` so that it's clear the angle is in degrees.